builds/jazz-build-module/aws-lambda-events-module.groovy

```diff
@@ -55,11 +55,11 @@ def updateKinesisResourceServerless(event_stream_arn){
   sh  "sed -i -- 's/#ResourceKinesisDisabled/Resource/g' ./policyFile.yml"
 }
 
-def getRoleArn(role_name) {
+def getRoleArn(role_name, credsId) {
   def role_arn
   try {
     def response = sh(
-      script: "aws iam get-role --role-name ${role_name} --profile cloud-api --output json",
+      script: "aws iam get-role --role-name ${role_name} --profile ${credsId} --output json",
```

Comment:
I can see we are still using cloud-api in this file. Please remove it

Reply:
Removed cloud-api and replaced with credsId